TailLight/driver.cpp

```diff
@@ -1,18 +1,33 @@
 #include "driver.h"
 
-/** Driver entry point.
-    Initialize the framework and register driver event handlers. */
+
+
 NTSTATUS DriverEntry(
     _In_ PDRIVER_OBJECT  DriverObject,
     _In_ PUNICODE_STRING RegistryPath
     )
+/*++
+    Routine Description:
+
+        Driver entry point.
+        Initialize the framework and register driver event handlers.
+
+    Arguments:
+
+        DriverObject - our WDM driver object.
+
+        RegistryPath - Taillight's registry key path.
+
+    Returns:
+
+        A success code if all steps worked, otherwise a failure code.
```

Comment:
Why are you bundling documentation changes for `DriverEntry` with this PR? This increases the size of the PR in an unnecessary way.

The documentation added for arguments and return-value is furthermore obvious, so just increases verbosity without adding any value.